devicesetup/src/main/java/io/particle/android/sdk/devicesetup/ParticleDeviceSetupLibrary.java

```diff
@@ -85,16 +93,38 @@ public IntentFilter buildIntentFilter() {
 
     /**
      * Start the device setup process.
+     *
+     * @deprecated Use {@link ParticleDeviceSetupLibrary#startDeviceSetup(Context, Class)}
```

Comment:
javadoc is nice but annotations are better. Please mark the method itself with @Deprecated